core/src/main/resources/hudson/model/ComputerSet/index.jelly

```diff
@@ -37,13 +37,13 @@ THE SOFTWARE.
       <div class="jenkins-app-bar__content">
         <h1>${%Manage nodes and clouds}</h1>
       </div>
-      <l:hasAdministerOrManage>
+      <j:if test="${h.hasAnyPermission(app, it.EXTENDED_READ, app.MANAGE)}">
```

Comment:
I suspect this should be `it`, normally needed for folders to work (not sure if you need it for ComputerSets though, but likely better to be safe?
```suggestion
      <j:if test="${h.hasAnyPermission(it, it.EXTENDED_READ, app.MANAGE)}">
```

Reply:
Ok, I thought about this, but wasn't sure.